src/main/java/com/hivemq/cli/commands/hivemq/script/ScriptGetCommand.java

```diff
@@ -0,0 +1,85 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hivemq.cli.commands.hivemq.script;
+
+import com.hivemq.cli.MqttCLIMain;
+import com.hivemq.cli.commands.hivemq.datahub.DataHubOptions;
+import com.hivemq.cli.commands.hivemq.datahub.OutputFormatter;
+import com.hivemq.cli.hivemq.scripts.GetScriptTask;
+import com.hivemq.cli.openapi.hivemq.DataHubScriptsApi;
+import com.hivemq.cli.rest.HiveMQRestService;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.tinylog.Logger;
+import picocli.CommandLine;
+
+import javax.inject.Inject;
+import java.util.concurrent.Callable;
+
+@CommandLine.Command(name = "get",
+                     description = "Get an existing script",
+                     synopsisHeading = "%n@|bold Usage:|@  ",
+                     descriptionHeading = "%n",
+                     optionListHeading = "%n@|bold Options:|@%n",
+                     commandListHeading = "%n@|bold Commands:|@%n",
+                     versionProvider = MqttCLIMain.CLIVersionProvider.class,
+                     mixinStandardHelpOptions = true)
+public class ScriptGetCommand implements Callable<Integer> {
+
+    @SuppressWarnings({"NotNullFieldNotInitialized", "unused"})
+    @CommandLine.Option(names = {"-i", "--id"}, required = true, description = "The id of the script")
+    private @NotNull String scriptId;
+
+    @SuppressWarnings("unused")
+    @CommandLine.Option(names = {"-f", "--field"},
+                        description = "Filter which JSON fields are included in the response")
+    private @Nullable String @Nullable [] fields;
+
+    @CommandLine.Mixin
+    private final @NotNull DataHubOptions dataHubOptions = new DataHubOptions();
+
+    private final @NotNull OutputFormatter outputFormatter;
+    private final @NotNull HiveMQRestService hiveMQRestService;
+
+    @Inject
+    public ScriptGetCommand(
+            final @NotNull HiveMQRestService hiveMQRestService, final @NotNull OutputFormatter outputFormatter) {
+        this.outputFormatter = outputFormatter;
+        this.hiveMQRestService = hiveMQRestService;
+    }
+
+    @Override
+    public @NotNull Integer call() {
+        Logger.trace("Command {}", this);
+
+        final DataHubScriptsApi scriptsApi =
+                hiveMQRestService.getScriptsApi(dataHubOptions.getUrl(), dataHubOptions.getRateLimit());
+
+        if (scriptId.isEmpty()) {
+            outputFormatter.printError("The script id must not be empty.");
+            return 1;
+        }
+
+        final GetScriptTask getScriptTask = new GetScriptTask(outputFormatter, scriptsApi, scriptId, fields);
+        if (getScriptTask.execute()) {
+            return 0;
+        } else {
+            return 1;
+        }
+    }
+    
```

Comment:
Missing toString (although I don't quite understand why we even need that in the first place haha)

(also missing in a few other script classes)